_automating-configurations/default-use-cases.md

```diff
@@ -0,0 +1,200 @@
+    "create_index.name": "my-nlp-index",
+    "create_index.settings.number_of_shards": "2",
+    "create_index.mappings.method.engine": "lucene",
+    "create_index.mappings.method.space_type": "l2",
+    "create_index.mappings.method.name": "hnsw",
+    "text_embedding.field_map.output.dimension": "1024",
+    "create_search_pipeline.pipeline_id": "default_model_pipeline"
+}
+```
+
+### Example overriding of params:
+
+```
+POST /_plugins/_flow_framework/workflow?use_case=semantic_search_with_cohere_embedding_query_enricher
+{
+    "create_connector.model" : "embed-multilingual-v3.0",
+    "create_ingest_pipeline.text_embedding.field_map.output": "book_embedding",
+    "create_index.name": "sparse-book-index"
+}
+```
+
+* In the example above, we are changing the Cohere model we want to use
+* The name of text_embedding processor output field
+* The name of the sparse index we create
+
+#### Additional Note:
+
+* It is important to note that every field listed in the defaults can be overridden by the user, so different use cases can have more flexibility over the sample index content and other important parameters in configuration
+
+### All Available Use Cases
+
+| Use Case Name                                        | Description                                                                                                                                                                                                                                                                       | required params | link to defaults |
+| ---------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------- | ---------------- |
+| bedrock-titan-embedding_model_deploy                 | Creates and deploys an Amazon Bedrock embedding model, defaulting to titan-embed-text-v1                                                                                                                                                                                          | AWS Credentials |[pre-set-defaults-file](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/bedrock-titan-embedding-defaults.json)|
+| bedrock-titan-multimodal_model_deploy                | Creates and deploys an Amazon Bedrock multimodal embedding model, defaulting to titan-embed-image-v1                                                                                                                                                                              | AWS Credentials |[pre-set-defaults-file](https://github.com/opensearch-project/flow-framework/blob/2.13/src/main/resources/defaults/bedrock-titan-embedding-defaults.json)                  |
```

Comment:
I think we can provide some cleaner direction on the required params. Either listing the exact fields that need to be in the JSON body, or specific examples somewhere for both overwriting "AWS Credentials", and overwriting "API Key`. It's not necessarily clear when linking to the defaults the precise set of fields required.

Reply:
Changed this